### PR TITLE
Weekend 4 Chitter Challenge

### PR DESCRIPTION
**Pull request by AndreaDiotallevi**

### CODE REVIEW

File: spec/features/see_peep_time_spec.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+feature "See peep's time" do
```
Comment (Hyan18) [inline spec/features/see_peep_time_spec.rb]: I like how you tested for time (using the Timecop gem)

File: spec/features/authentication_spec.rb [hunk 2]
```diff
@@ -0,0 +1,41 @@
+feature "Authentication" do
+  scenario "A maker can sign in" do
+    sign_up
```
Comment (Hyan18) [inline spec/features/authentication_spec.rb]: Good use of helper methods

File: spec/features/registration_spec.rb [hunk 3]
```diff
@@ -0,0 +1,35 @@
+feature "Registration" do
+  scenario "A maker can sign up" do
+    sign_up
+
+    expect(page).to have_content "Welcome Angea89"
+  end
+
+  scenario "The maker's email must be unique" do
```
Comment (Hyan18) [inline spec/features/registration_spec.rb]: Good tests to cover edge cases of unique email and username

File: spec/features/view_peeps_spec.rb [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    sign_up
+    sign_in
+    post_peep
+
+    visit "/peeps/new"
+    fill_in "content", with: "Second peep"
+    click_button "Post"
+
+    expect(current_path).to eq "/peeps"
+    expect("Second peep").to appear_before("First peep")
```
Comment (Hyan18) [inline spec/features/view_peeps_spec.rb]: Like the use of the 'orderly' gem to make the test more readable.
There is a way to test using CSS' nth child syntax without the use of a gem.

File: views/peeps/index.erb [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+<h2>
+  <%= flash[:notice] %>
+</h2>
+
+<% if @maker %>
+  <h2>Welcome <%= @maker.username %></h2>
+<% end %>
+
+<ul>
+  <% @peeps.reverse_each do |peep| %>
```
Comment (Hyan18) [inline views/peeps/index.erb]: Could put the reversing logic in the controller instead of the view (i.e using ORDER BY in the SQL query)
